index.html

```diff
@@ -638,6 +638,23 @@ <h2>A project for first-time contributions</h2>
 
         </div>
         <!-- ________ Brener's card END ________  -->
+        <!-- ________ Contributor's card START ________  -->
+        <div class="card">
+
+            <p class="name">Ania Mierzwinska</p>
+            <p class="contact"><i class="fa github"></i> <a href="https://github.com/vilanelle/" target="_blank">@vilanelle</a></p>
```

Comment:
The GitHub icon class is not written correctly. I think it should be `fa fa-github` but it's a good idea to check. Please change this before I can merge your card.